examples/testing-module/moduleframework-cmd

```diff
@@ -1 +1 @@
-../../moduleframework/bashhelper.py
\ No newline at end of file
+../../tools/bashhelper.py
```

Comment:
This line requires docs update: api/bashhelper.rst:4:.. automodule:: moduleframework.bashhelper